frontend/src/hooks/useRouterFilter.ts

```diff
@@ -51,8 +51,8 @@ export const useRouterFilter = <T extends BasePropertyFilter>(
   React.useEffect(() => {
```

Comment:
Goeme and I found this. The route was not being changed after the filter was applied on the landing page. Seems to be working now